src/app/shared/forms/planet-tag-input-dialog.component.html

```diff
@@ -58,6 +58,7 @@
         <planet-tag-input-toggle-icon *ngIf="tag.subTags.length > 0" (click)="toggleSubcollection($event,tag._id)" [isOpen]="subcollectionIsOpen.get(tag._id)"></planet-tag-input-toggle-icon>
         <span class="toolbar-fill"></span>
         <button mat-stroked-button *ngIf="isUserAdmin" (click)="editTagClick($event,tag)">Edit</button>
+        <button mat-stroked-button *ngIf="isUserAdmin" [disabled]="tag.subTags.length > 0" (click)="deleteTag($event,tag)">Delete</button>
```

Comment:
The `span` with the tooltip should be included so when the button is disabled there is an indication to the user why it is disabled.